J/JuliaInterpreter/Compat.toml

```
@@ -1,20 +1,39 @@
-["0-0.7"]
-julia = "1"
+["0 - 0.7"]
+julia = "1.0"
```

We want to prevent all old versions to install on Julia v1.8.  So this should be
```suggestion
julia = "1.0 - 1.7"
```